### DISCUSSION TITLE
nemo - when delete key pressed for folders in /usr/share/, nothing happens, a feedback should be presented to tell the user

### DISCUSSION THREAD

**Discussion by skygunner** [linuxmint/discussions#661] [Ideas]
Currently for selected folders in /usr/share,
press delete key,  there's nothing happens , no matter its normal user or opened the folder as root. 
This confuses the end user. If the file manager could give any feedback telling permission required would be good. Or pop up dialog to accept password for root. 

version：6.2.8

**Comment by Jopp-gh** [comment]
Well, directories outside the user space (outside your user account) are system relevant and therefore **off limits** for common users. 

Mint gives you all the tools you need so you don't have to worry and mess with system files, with tools like Mint's preferences or installer apps or other utilities. 

"Administrators" are able to configure and troubleshoot difficult and complex technical tasks. 
But performing these tasks requires a proper training. 

So, real Administrators don't need "cute" user-interface suggestion dialogs.
